### PR TITLE
Added code and tests the read record functionality

### PR DESCRIPTION
**Pull request by rishit13**
This PR contains the addition of `read inventory record `functionality. 

1. Functionality

Added `get_inventory_records:` to the routes.py file. This functionality takes `product ID` and `condition  ` as arguments from the user  and search the inventory database for the record and return the product id, name, quantity, condition, restock  level and reorder quantity. 

Here product d is passed in the URL and the condition is passed as part of the payload. These are used by the find function (present in the models.py file) to search for the particular record. 

2. Added the following test cases : 

- test_read_records
- test_read_non_existent_records 

Increased the routes.py code coverage from 95% to 100%.

### CODE REVIEW

**Review by sakshambassi — APPROVED** [review]
awesome, looks good

File: service/models.py [hunk 1]
```diff
@@ -135,12 +135,13 @@ def find(cls, by):
...
-    @classmethod
-    def find_by_name(cls, name):
-        """Returns all Inventories with the given name
-
-        Args:
-            name (string): the name of the Inventories you want to match
-        """
-        logger.info("Processing name query for %s ...", name)
-        return cls.query.filter(cls.name == name)
+    #uncomment for sprint 2
```
Comment (aayushagrawal135) [inline service/models.py:138]: In commented code is unused, can consider removing it.

**Review by richa15997 — APPROVED** [review]

File: service/models.py (hunk 1 above)
Comment (rishit13) [reply service/models.py:138]: Yeah, was thinking about it, but we will require it in the next sprint, and this ensure a code coverage of 91% for models.py

Comment (aayushagrawal135) [reply service/models.py:138]: Alright!

**Review by aayushagrawal135 — APPROVED** [review]

**Pull request merged**